### PR TITLE
config_env_changes: Added environment variables for configurable valu…

### PR DESCRIPTION
**Pull request by chirag-zymr**
…es like slack token and email values.

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -3,24 +3,39 @@ services:
...
     image: 'mongo:3.4.9'
     container_name: mongodb
+    restart: always
     ports:
       - '27017:27017'
     volumes:
       - data:/data/db
+      - mongologs:/var/log/mongodb
     environment:
       MONGO_INITDB_DATABASE: VisitorManagement
```
Comment (nikunj-zymr) [inline docker-compose.yaml]: db_zvisitor

**Review by nikunj-zymr — DISMISSED** [review]

File: Jenkinsfile [hunk 2]
```diff
@@ -13,17 +13,54 @@ pipeline {
...
+                
+              sh '''
+                'ls'
+                rm -rf .env 
+                ls -la
+                cat >> .env <<EOL
+                MAIL_PERSONAL:'$MAIL_PERSONAL'
+                ADMIN_MAIL:'$ADMIN_MAIL'
+                ADMIN_PASSWORD:'$ADMIN_PASSWORD'
+                MONGODB_USERNAME:'$MONGODB_USERNAME'
```
Comment (chirag-zymr) [inline Jenkinsfile]: @Akshaykumar-Hingarajiya this seems duplicate line

File: docker-compose.yaml [hunk 3]
```diff
@@ -3,24 +3,47 @@ services:
...
+      USER_APP_PORT: 8080
+      MAIL_PERSONAL: z-visitor
+      ADMIN_MAIL: $ADMIN_MAIL
+      ADMIN_PASSWORD: $ADMIN_PASSWORD
+      SLACK_TOKEN: $SLACK_TOKEN
+      SLACK_USERNAME: $SLACK_USERNAME
+      MAIL_USERNAM: $MAIL_USERNAM
+      MAIL_PASSWORD: $MAIL_PASSWORD
+      MONGODB_USERNAME: $MONGODB_USERNAME
+      MONGODB_PASSWORD: $MONGODB_PASSWORD
```
Comment (sitanshu-zymr) [inline docker-compose.yaml:41]: Good

Comment (jagdish-zymr) [reply docker-compose.yaml:41]: @Akshaykumar-Hingarajiya  As we use the name volume so add volume path details in the ReadMe file.

**Review by jagdish-zymr — COMMENTED** [review]
looks good

File: docker-compose.yaml (hunk 3 above)
Comment (Akshaykumar-Hingarajiya) [reply docker-compose.yaml:41]: Done.

**Review by nikunj-zymr — APPROVED** [review]

**Pull request merged**